# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.cpp
# Commit: 3daeacd08336
# Full Hash: 3daeacd0833682a474d9bd1adc8817474a7602be
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 6: Move NotificationObserver functions to NotificationUtils r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227691
# ==============================================================================

diff -r a497ddb030f6 -r 3daeacd08336 dom/ipc/ContentParent.cpp
--- a/dom/ipc/ContentParent.cpp	Thu Nov 07 17:32:05 2024 +0000
+++ b/dom/ipc/ContentParent.cpp	Thu Nov 07 17:32:05 2024 +0000
@@ -113,7 +113,7 @@
 #include "mozilla/dom/MediaController.h"
 #include "mozilla/dom/MemoryReportRequest.h"
 #include "mozilla/dom/MediaStatusManager.h"
-#include "mozilla/dom/Notification.h"
+#include "mozilla/dom/notification/NotificationUtils.h"
 #include "mozilla/dom/PContentPermissionRequestParent.h"
 #include "mozilla/dom/PCycleCollectWithLogsParent.h"
 #include "mozilla/dom/ParentProcessMessageManager.h"
@@ -4725,7 +4725,7 @@
   if (!ValidatePrincipal(aPrincipal)) {
     LogAndAssertFailedPrincipalValidationInfo(aPrincipal, __func__);
   }
-  Unused << Notification::RemovePermission(aPrincipal);
+  Unused << notification::RemovePermission(aPrincipal);
   return IPC_OK();
 }
 
@@ -4738,7 +4738,7 @@
   if (!ValidatePrincipal(aPrincipal)) {
     LogAndAssertFailedPrincipalValidationInfo(aPrincipal, __func__);
   }
-  Unused << Notification::OpenSettings(aPrincipal);
+  Unused << notification::OpenSettings(aPrincipal);
   return IPC_OK();
 }
 